pkg/resourcemanager/rediscaches/redis/rediscache_reconcile.go

```diff
@@ -49,6 +49,7 @@ func (rc *AzureRedisCacheManager) Ensure(ctx context.Context, obj runtime.Object
 			}
 			instance.Status.Message = resourcemanager.SuccessMsg
 			instance.Status.State = string(newRc.ProvisioningState)
+			instance.Status.Output = *newRc.StaticIP
```

Comment:
I got a panic on this line when trying to create a rediscache with a shard count (but without the vnet configured). We might need to check for StaticIP not nil before assigning

Reply:
should be resolved now